### DISCUSSION TITLE
Welcome to nyxtext Discussions!

### DISCUSSION THREAD

**Discussion by parazeeknova** [parazeeknova/nyxtext#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rohankishore** [comment]
Hey! I see you're making a text editor with Tkinter. As you've already seen, I have some experience building IDEs using PyQt6/Tkinter. Aura Text (my ide project) was powered by Tk in its starting days. 

https://github.com/rohankishore/Aura-Text/releases/tag/v1.7

I thought this might be helpful for you. Also, I'd recommend you to start learning PyQt too since you'll have much better editing interface like QScintilla.

P.S: I'll be writing VITEEE this year so if I get lucky, we might meet in the campus :)

**Reply by parazeeknova** [reply]
Hi @rohankishore 👋,

Thanks for your interest in **NyxText**! We're currently using a combination of **_Tkinter and CustomTkinter_**. We're in the very early stages of development (only 3 weeks in!) and still have a long way to go. We started with **_Tkinter_** as the base and will continue using it until it reaches its limits for our project. While PyQt6 was an option, we were already invested by then.

**Here's a spoiler!** We'll soon be creating another project based on PyQt6 and other Qt components for a complete IDE. After exploring options, QScintilla emerged as the perfect fit for text editing in our next project .

I checked out your project "Aura-Text." It's amazing!  While I haven't fully learned PyQt6 yet, I can't comment on the codebase in detail. However, from what I've seen, it seems very powerful and packed with features that we might also consider implementing in future versions of NyxText & future IDE .

P.S. Best of luck with your exams!  In the meantime, if you have any questions about text editor development, feel free to ask! We're also open to contributions, if you're interested in getting involved in an early-stage project as its our **first one**.

**Comment by rohankishore** [comment]
Yeah good luck with your IDE project. I'll be more than happy to contribute.

**Reply by parazeeknova** [reply]
Great to hear, We really appreciate your willingness to help out.
